core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -686,7 +690,7 @@ query.timeout.sec=60
 #pg.character.store.pool.capacity=64
 #pg.connection.pool.capacity=64
 #pg.password=quest
-#pg.user=admin
+#pg.user=adminBas
```

Comment:
Do we need this change? The default seems to be `admin`.

Reply:
🤦

Reply:
thank you! 🦅 👁️ !